src/lib/sort/sort.md

```diff
@@ -0,0 +1,31 @@
+The `mdSort` and `md-sort-header` are used, respectively, to add sorting behavior, state, and display
+to tabular data.
+
+<!-- example(sort-overview) -->
+
+# Adding sort to table headers
+
+To add sorting behavior and styling to a set of table headers, add the `<md-sort-header>` component
+to each header and provide an `id` that will identify it. These headers should be contained within a
+parent element with the `mdSort` directive, which will emit an `mdSortChange` event when the user
+ triggers sorting on the header.
+
+Users can trigger the sort header through a mouse click or keyboard action. When this happens, the
+`mdSort` will emit an `mdSortChange` event that contains the ID of the header triggered and the
+direction to sort (`asc` or `desc`).
+
+### Changing the sort order
+
+By default, a sort header starts its sorting at `asc` and then `desc`. Triggering the sort header
+after `desc` will remove sorting.
+
+To reverse the sort order for all headers, set the `mdSortStart` to `desc` on the `mdSort` directive.
+To reverse the order only for a specific header, set the `start` input only on the header instead.
+
+To prevent the user from removing sort altogether after it is applied, set `mdSortDisableClear` to
```

Comment:
"To prevent the user from clearing the sort state from an already sorted column"